corehq/apps/app_manager/management/commands/audit_case_search_config.py

```diff
@@ -58,9 +61,18 @@ def handle(self, **options):
 
         result_domains = {b.domain for b in results}
```

Comment:
just out of curiosity, do you know why we're using `b` here?

Reply:
I don't know. I suppose it would make as much or more sense to use `r` for `result`.